src/HierarchyAnalysisAlgorithm.cc

```diff
@@ -287,17 +305,18 @@ void HierarchyAnalysisAlgorithm::EventAnalysisOutput(const LArHierarchyHelper::M
 
                 // Save the track score, getting the appropriate metadata
                 // see e.g. https://github.com/PandoraPFA/larpandora/blob/develop/larpandora/LArPandoraInterface/LArPandoraOutput.cxx#L325 for similar
-                const auto& properties = pPfo->GetPropertiesMap();
-                float trackScore = -1.;
+                const auto &properties = pPfo->GetPropertiesMap();
+                float trackScore{-1.f};
                 const auto iterTrackScore(properties.find("TrackScore"));
-                if ( iterTrackScore != properties.end() ) {
+                if (iterTrackScore != properties.end())
+                {
                     trackScore = iterTrackScore->second;
                 }
-                trackScoreVect.emplace_back( trackScore );
+                trackScoreVect.emplace_back(trackScore);
 
-		// Define isShower based on track score
-		const int isShower = (trackScore > m_minTrackScore) ? 0 : 1;
-		isShowerVect.emplace_back(isShower);
+                // Define isShower based on track score
+                const int isShower = (trackScore >= m_minTrackScore) ? 0 : 1;
```

Comment:
Good catch! I double checked and confirmed that the MvaPfoCharacterisation algorithm also uses >= https://github.com/PandoraPFA/LArContent/blob/master/larpandoracontent/LArTrackShowerId/MvaPfoCharacterisationAlgorithm.cc#L283